src/main/java/com/easypost/net/Requestor.java

```diff
@@ -57,89 +57,10 @@ protected enum RequestMethod {
 
     private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
     private static final String CUSTOM_URL_STREAM_HANDLER_PROPERTY_NAME = "com.easypost.net.customURLStreamHandler";
-    private static int connectTimeoutMilliseconds = Constant.DEFAULT_CONNECT_TIMEOUT_MILLISECONDS;
```

Comment:
Please put these timeout vars and functions back, they were in the correct place.

Reply:
resolved